### PR TITLE
kicker reward exceeds borrower penalty fix

### PR DESCRIPTION
**Pull request by ith-harvey**
## Description

* Regression test shows failure kicker reward exceeding borrower penalty. @mattcushman 's analysis is that this amount is not to worry about since it was `0.000000000000000001`... a greater than with diff was added to resolve the issue.

## Tasks

- [ ] Changes to protocol contracts are covered by unit tests executed by CI.
- [ ] Protocol contract size limits have not been exceeded.
- [ ] Gas consumption for impacted transactions have been compared with the target branch, and nontrivial changes cited in the _Impact_ section above.
- [ ] Scope labels have been assigned as appropriate.
- [ ] Invariant tests have been manually executed as appropriate for the nature of the change.

### CODE REVIEW

File: tests/forge/invariants/base/LiquidationInvariants.t.sol [hunk 1]
```diff
@@ -130,15 +131,20 @@ abstract contract LiquidationInvariants is BasicInvariants {
...
         uint256 kickerReward    = IBaseHandler(_handler).kickerReward();
 
         console.log("Borrower Penalty -->", borrowerPenalty);
         console.log("Kicker Reward    -->", kickerReward);
 
-        require(kickerReward <= borrowerPenalty, "Auction Invariant A8");
+        greaterThanWithinDiff(
+            borrowerPenalty,
+            kickerReward,
+            1e4,
```
Comment (EdNoepel) [inline tests/forge/invariants/base/LiquidationInvariants.t.sol]: Why 1e4 here?  I've never seen a diff more than 1 due to rounding error.

Comment (ith-harvey) [reply tests/forge/invariants/base/LiquidationInvariants.t.sol]: @mattcushman just made a call here... I think we could try and reduce it to see if the error pops up again if we run on pavarti

Comment (EdNoepel) [reply tests/forge/invariants/base/LiquidationInvariants.t.sol]: I prefer @grandizzy 's approach of rounding the calculation, as implemented here: https://github.com/ajna-finance/contracts/commit/97f62b53792f48ba684501c6718107f7d9fde6af

**Comment by ith-harvey** [comment]
Resolved in https://github.com/ajna-finance/contracts/pull/1006

**Pull request closed without merging**